evals/features/cve/test_suggest_impact.py

```diff
@@ -112,7 +112,11 @@ async def suggest_impact(cve_id: CVEID) -> SuggestImpactModel:
     SuggestImpactCase("CVE-2024-53232", "MODERATE", 4.4),
     SuggestImpactCase("CVE-2025-23395", "MODERATE", 6.8),
     SuggestImpactCase("CVE-2025-5399", "MODERATE", 4.3),
-    # TODO: add more cases
+    SuggestImpactCase("CVE-2025-9573", "IMPORTANT", 7.2),
+    SuggestImpactCase("CVE-2023-39326", "MODERATE", 7.5),
+    SuggestImpactCase("CVE-2023-53693", "MODERATE", 5.5),
+    SuggestImpactCase("CVE-2025-12735", "CRITICAL", 9.8),
+    SuggestImpactCase("CVE-2025-59840", "IMPORTANT", 8.1),
```

Comment:
I would keep the list sorted so that we can easily add new cases (while avoiding duplicated entries).

I can see that one entry had already been misplaced.  These were added manually.  The `process_cve_feedback_form.py` script now generates these lists already sorted.